src/ProcessManagerBundle/DependencyInjection/Configuration.php

```diff
@@ -42,6 +42,7 @@ public function getConfigTreeBuilder(): TreeBuilder
             ->children()
                 ->scalarNode('driver')->defaultValue(CoreShopResourceBundle::DRIVER_PIMCORE)->end()
                 ->scalarNode('log_directory')->defaultValue('%kernel.logs_dir%')->end()
+                ->booleanNode('keep_logs')->defaultValue(true)->end()
```

Comment:
we should also add a default configuration on how long we keep the logs and have a maintenance job clearing them automatically.

Reply:
Pimcore maintence task created. Default time to keep the data is 7 days.